operations/reservations.md

```diff
@@ -524,6 +524,7 @@ Adds the specified reservations as a single group. If `GroupId` is specified, ad
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Identifier` | string | optional | Identifier of the reservation within the transaction. |
+| `ChannelNumber` | string | optional | Number of the reservation within the Channel \(i.e. OTA, GDS, CRS, etc\) in case the reservation group originates there \(e.g. Booking.com confirmation number\). |
```

Comment:
Sort it same as in table of reservatin properties

Reply:
What do you mean?

Reply:
Ordering of fields in this table should be same as their ordering in table describing properties of reservation entity.